src/EventStore.Storage/Storage/EpochManager/EpochManager.cs

```diff
@@ -223,8 +219,8 @@ public class EpochManager : IEpochManager {
 
 			Log.Debug("=== Writing E{epochNumber}@{epochPosition}:{epochId:B} (previous epoch at {lastEpochPosition}). L={leaderId:B}.",
 				epochNumber, epoch.EpochPosition, epochId, lastEpochPosition, epoch.LeaderInstanceId);
-
-			_bus.Publish(new SystemMessage.EpochWritten(epoch));
+			//todo - publish this
```

Comment:
This `todo` can be removed as we are publishing this message from all the `WriteNewEpoch` callers.